get-thin-edge_io.sh

```diff
@@ -270,6 +280,13 @@ main() {
         PURGE_OLD_PACKAGES=1
     fi
 
+    # Ignore plugins for older versions
+    if dpkg --compare-versions "$VERSION" lt "0.10.0"; then
+        log "ignore c8y-firmware-plugin and c8y-remote-access-plugin as they are not supported in <= 0.10.0"
+        C8Y_FIRMWARE_PLUGIN=
+        C8Y_REMOTE_ACCESS_PLUGIN=
+    fi
```

Comment:
Longer term it would be good to find something less error prone.

Reply:
Or, a `cloud-specific device management component` that will have all the related components.